### ISSUE TITLE
feedback bitcoin-infinity

### ISSUE THREAD

**Issue by Drehschloss** [bitcoin/bitcoin#10066]
Hi together,
and congratulation!
 I just compiled your new patch, which I could download from github.com. 
It did very well.
 But I got a warning about comparison of signed and unsigned int in the 
sourcefile miner.cpp (see attachement).
 The compilation was for version 14.0 and I use linux on a x64 machine.
 One integer expression is something like "int64_t"

Changing line 152 in your file miner.h:
    uint64_t nBlockSigOpsCost;
to
    int64_t nBlockSigOpsCost;
makes the compiler warning-message to disappear as I could notice.

Have a nice day and thank you for the patch!

![make_warning](https://cloud.githubusercontent.com/assets/26636524/24271706/6d2895bc-101a-11e7-8876-0bf713c399f3.png)

**Comment by paveljanik** [comment]
We do not have such code in our tree. Wrong repository. Let's close this.

**Comment by Drehschloss** [comment]
Hi paveljanik,

first thanks for your answer.
Sorry, if this is not your code.
But what's this? (See attachment)
I am using bitcoin core since version 0.12.1 and until now I took the ready compiled programs (for linux 64bit).
Thus I compiled the "infinity-patch" (for version 0.14.0) nowadays for the first time and I saw a warning resulting from comparison of variables "signed int" and "unsigned int". 
This may be harmless, but it can give problems if it is possible to change the contents of such variables at runtime (what I do not know).

Cheers,

Drehschloss


Thursday, march 23rd 2017, 23:48:05 paveljanik wrote:
> We do not have such code in our tree. Wrong repository. Let's close this.
> 
> 
>
